README.md

```diff
@@ -48,11 +48,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [hello Microverse] <a name="about-project"></a>
+# 📖 [Portfolio] <a name="about-project"></a>
 
-> hello Microverse project 
+> Portfolio: setup and mobile first 
 
-**[hello Microverse]** is a... html project with hello Microverse and change the colour
+**[: Portfolio setup and mobile first]** Portfolio I can put on it all my work and personal information
```

Comment:
- [ ] please Kindly fix the MIT link of your README.md file. It shows a 404 error page when it is clicked.
we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.